### DISCUSSION TITLE
快捷切换联网更新主页和本地文件主页

### DISCUSSION THREAD

**Discussion by xianhongtao** [Meloong-Git/PCL#1553] [讨论]
现在联网更新的主页和用户自己写的主页不能共存，导致了使用不便，希望能加入快捷切换的按钮来切换各个主页来源或使多个主页以一定方式共存

**Comment by WorldHim** [comment]
额，下次交提案发 issue，我帮你发到 issue 上了。#1154

**Reply by WTP016-CN** [reply]
issue 号写错了(小声bb

**Reply by MoYuan-CN** [reply]
正确的应为 #1554 
~~并且被拒绝了~~

**Reply by xianhongtao** [reply]
难受

**Comment by WForst-Breeze** [comment]
进一步讨论请移步 #1554
